staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -49,23 +48,19 @@ type Backoff struct {
 	maxJitterFactor float64
 }
 
-func NewFakeBackOff(initial, max time.Duration, tc *testingclock.FakeClock) *Backoff {
```

Comment:
you can't move these to a _test file and preserve the exported API for things outside this package

Reply:
is not preserved, is it?

Reply:
"exported" symboles in _test files don't get exported outside the package

Reply:
yeah, but I mean, is a cosmetic problem, isn;t it?